### PR TITLE
custom configs samples for bundles

### PR DESCRIPTION
**Pull request by david-truong**
@gamerson @Ithildir wanna take a look and offer any suggestions?

### CODE REVIEW

**Comment by gamerson** [comment]
1) What are the differences between "dev" and "local"?  Do we need both?

2) Is there a way to avoid having that binary jar that is specify to tomcat in our liferay workspace?  can't we just use the default msyql version that the portal auto-downlaods as the default and then if the user wants to switch to mariadb they can?  We could even add that command to blade.  

3) Is it standard practice to put username/password into properties and commit them to VCS? 

4) By putting these files in here by default we are assuming the user has created their databases with these specific names (lportal_dev, etc).  Is there a way we can keep the default workspace build that doesn't use these properties (or configs) and then make it easy for the user to figure out how to "enable" these configs... once they have run the Portal Setup Wizard (and configured their database)?  That is the normal workflow right?

**Comment by david-truong** [comment]
@gamerson 

1) local is a developer’s machine, dev is the dev server.

2) It’s just an example so we shouldn’t include it in the release.

3) No I doubt they will do it this way so maybe we should change the samples to be a little more realistic.

4) Again these are just examples… so we can just update it to be a little more realistic if you want.

I think we should create a new project called liferay-workspace-samples and move all these samples to that project.  Then liferay-workspace will become the new releases branch.

**Comment by gamerson** [comment]
1) I see, so dev is like a shared dev server that dev team is using in the office
2) got it, yes we need to show people how this type of configuration could be setup in workspace
3) can you reach out to the GS team or Josef and see how they do it?
4) right that makes sense, I have to remember that what is in liferay-workspace isn't what is "released" 

Regarding the separate repo, we already have "blade" repo.  Maybe we add the "workspace with samples" there?  So we don't have to have yet one more repo to keep in sync?  And then liferay-workspace will be just "pure" and in blade repo we have our "workspace"  version.

File: configs/dev/portal-ext.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#
+# MySQL
+#
+jdbc.default.driverClassName=org.mariadb.jdbc.Driver
+jdbc.default.url=jdbc:mysql://localhost/lportal_dev?useUnicode=true&characterEncoding=UTF-8&useFastDateParsing=false
+jdbc.default.username=root
+jdbc.default.password=
+
+
```
Comment (Ithildir) [inline configs/dev/portal-ext.properties]: Shouldn't we just do `include-and-override=portal-developer.properties`?

**Comment by gamerson** [comment]
@david-truong looks like the build failed because of the plugins-sdk ivy download.  Can we tell ivy to spit out its download output so travis doesn't kill the build?

**Comment by david-truong** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by david-truong** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/david-truong/liferay-workspace/compare/7c05cc5~3...7c05cc5) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**